draft-ietf-quic-transport.md

```diff
@@ -2397,12 +2397,13 @@ transmission of CONNECTION_CLOSE frames to validated addresses or drop packets
 without response if the response would be more than three times larger than the
 received packet.
 
-After receiving a CONNECTION_CLOSE frame, endpoints enter the draining state.
-An endpoint that receives a CONNECTION_CLOSE frame MAY send a single packet
-containing a CONNECTION_CLOSE frame before entering the draining state, using a
-CONNECTION_CLOSE frame and a NO_ERROR code if appropriate.  An endpoint MUST NOT
-send further packets, which could result in a constant exchange of
-CONNECTION_CLOSE frames until the closing period on either peer ended.
+After receiving a CONNECTION_CLOSE frame, or an invalid packet that appears to
+contain a CONNECTION_CLOSE frame, endpoints enter the draining state.  An
```

Comment:
Looks like Github swallowed my previous comment on this line:
How would you even parse an invalid packet? What does it mean to "appear to contain" a frame?

Reply:
A packet might "appear to contain a frame" if it contains the truncated beginning of a frame.

Reply:
"or an invalid packet that appears to contain a CONNECTION_CLOSE_ frame," is a pretty big category of potential issues.  Does this need to be called out as separate from the general invalid packet case?

Reply:
I think that we'll just go with #3231 and avoid having to worry about this definition.